serialization/ruby/json/lib/json_serialization_writer.rb

```diff
@@ -1,7 +1,68 @@
 require 'microsoft_kiota_abstractions'
+require 'time'
+require 'json'
+require "uuidtools"
 
 module MicrosoftKiotaSerialization
     class JsonSerializationWriter
         include MicrosoftKiotaAbstractions::SerializationWriter
+        @writer = Hash.new()
+        def write_string_value(key, value)
+            if !key
+                raise ArgumentError.new "no key included in write_string_value(key, value)"
```

Comment:
in the other serialization writers I do allow for no key to be passed to write a scalar value. This way I can reuse those method to write the values of a collection of scalars.